test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/AbstractService.java

```diff
@@ -77,8 +81,7 @@
     static final String ZOOKEEPER_OPERATOR_IMAGE = System.getProperty("zookeeperOperatorImage", "pravega/zookeeper-operator:latest");
     static final String IMAGE_PULL_POLICY = System.getProperty("imagePullPolicy", "Always");
     private static final String DOCKER_REGISTRY =  System.getProperty("dockerRegistryUrl", "");
-    private static final String PRAVEGA_VERSION = System.getProperty("imageVersion", "latest");
-    private static final String PRAVEGA_BOOKKEEPER_VERSION = System.getProperty("pravegaBookkeeperVersion", PRAVEGA_VERSION);
```

Comment:
Similar to what @shrids has commented, why do we need to change these versions to enable Pravega operator `0.4.0` to work?

Reply:
Because these changes have already been incorporated in the operator 0.4.0 code and they need to be reflected in the CRD being created from the System Tests framework in order to work.

Reply:
This removes the ability to specify a specific bookkeeper version and locks us into a certain version, which is not correct. Have you tested if your changes enable running of system tests with other operator versions as well?